CIP-SUBLEDGER/README.md

```diff
@@ -0,0 +1,144 @@
+---
+CIP: ?
+Title: A Subledger in Plutus Enabling Native-Asset Seizure for Regulatory Compliance
+Status: Proposed
+Category: Ledger
+Authors:
+    - Micah Kendall <micah@butane.dev>
+Implementors: [Micah Kendall, <you?>]
+Discussions:
+    - https://github.com/cardano-foundation/CIPs/pull/?
+Created: 2024-05-30
+License: CC-BY-4.0
+---
+
+# CIP \*\*\*: A Subledger in Plutus Enabling Native-Asset Seizure for Regulatory Compliance
+
+## Abstract
+We aim to establish a simple standard to ensure native assets can exist in a locked smart contract system, referred to as the "subledger". This subledger will enable functionality defined in the minting policy to seize assets.
+
+This design preserves the benefits of Cardano's native assets by enabling the reuse of most EUTxO patterns. It is significant that DEXs or other Plutus applications can be easily adapted to function within the subledger environment.
+
+# Motivation: why is this CIP necessary?
+
+A solution is required to enable asset seizure such as revoking USDC. We attempt to provide one in an EUTXO-harmonious style, for the sake of developer & user experience. 
+
+# Specification
+## Objectives
```

Comment:
```suggestion
### Objectives
```
(correcting header level for standard CIP outline)

Likewise, please indent all further headers in your specification as necessary so that it's organised properly within the H2 _Specification_ until the next H2 section _Rationale_.